app/src/main/res/layout/routes_activity.xml

```diff
@@ -70,15 +70,15 @@
                 app:layout_constraintTop_toTopOf="parent" />
 
             <TextView
-                android:onClick="onClickFrom"
-                android:clickable="true"
                 android:id="@+id/fromText"
                 android:layout_width="303dp"
                 android:layout_height="39dp"
                 android:layout_marginTop="8dp"
                 android:background="#29FFFFFF"
+                android:clickable="true"
                 android:ems="10"
                 android:inputType="textPersonName"
+                android:onClick="onClickFrom"
                 android:padding="8dp"
                 android:text="Name"
```

Comment:
Text should be written in strings.xml and referenced here, it will be easier that way when we support french text, we will just need to add the text in the french strings file with the same id.

Reply:
![image](https://user-images.githubusercontent.com/34378319/77014700-49ce3000-6949-11ea-9774-6294533d4d9c.png)

Reply:
This is actually not part of what I did I'm not sure how the PR sees it as a new line.. But whoever worked on it is actually retrieving it by ID and replacing name so I think it should be fine